config/default/constants.js

```diff
@@ -20,97 +20,108 @@
+ * @property {number} FEES.DAPP_REGISTRATION	- Fee for registering as a dapp.
+ * @property {number} FEES.DAPP_WITHDRAWAL	- Fee for registering as a dapp.
+ * @property {number} FEES.DAPP_DEPOSIT	- Fee for registering as a dapp.
  // TODO: Needs additional check to revise max payload length
  // for each transaction type for consistency
  // FYI: https://lisk.io/documentation/the-lisk-protocol/blocks {Block Payload Section}
- * @property {number} maxPayloadLength - Maximum transaction bytes length for 25 transactions in a single block.
- * @property {number} maxPeers - Maximum number of peers allowed to connect while broadcasting a block.
- * @property {number} maxSharedTxs - Maximum number of in-memory transactions/signatures shared accros peers.
- * @property {number} maxTxsPerBlock -	Maximum Number of transactions allowed per block.
- * @property {number} minBroadhashConsensus - Minimum broadhash consensus(%) among connected {maxPeers} peers.
- * @property {string[]} nethashes - For mainnet and testnet.
- * @property {number} constants.normalizer - Use this to convert LISK amount to normal value.
- * @property {Object} rewards - Object representing LSK rewards milestone.
- * @property {number[]} rewards.milestones - Initial 5, and decreasing until 1.
- * @property {number} rewards.offset - Start rewards at block (n).
- * @property {number} rewards.distance - Distance between each milestone.
- * @property {number} totalAmount - Total amount of LSK available in network before rewards milestone started.
- * @property {number} unconfirmedTransactionTimeOut - Expiration time for unconfirmed transaction/signatures in transaction pool.
- * @property {number} expiryInterval - Transaction pool expiry timer in milliseconds
+ * @property {number} MAX_PAYLOAD_LENGTH - Maximum transaction bytes length for 25 transactions in a single block.
+ * @property {number} MAX_PEERS - Maximum number of peers allowed to connect while broadcasting a block.
+ * @property {number} MAX_SHARED_TRANSACTIONS -	Maximum number of in-memory transactions/signatures shared accros peers.
+ * @property {number} MAX_TRANSACTIONS_PER_BLOCK -	Maximum Number of transactions allowed per block.
+ * @property {number} MAX_VOTES_PER_TRANSACTION - The maximum number of votes allowed in transaction type(3) votes.
+ * @property {number} MIN_BROADHASH_CONSENSUS - Minimum broadhash consensus(%) among connected {MAX_PEERS} peers.
+ * @property {number} MULTISIG_CONSTRAINTS - Description needed
+ * @property {number} MULTISIG_CONSTRAINTS.MIN.MINIMUM - Description needed
+ * @property {number} MULTISIG_CONSTRAINTS.MIN.MAXIMUM - Description needed
+ * @property {number} MULTISIG_CONSTRAINTS.LIFETIME.MINIMUM - Description needed
+ * @property {number} MULTISIG_CONSTRAINTS.LIFETIME.MAXIMUM - Description needed
+ * @property {number} MULTISIG_CONSTRAINTS.KEYSGROUP.MIN_ITEMS - Description needed
+ * @property {number} MULTISIG_CONSTRAINTS.KEYSGROUP.MAX_ITEMS - Description needed
```

Comment:
@ManuGowda can you provide descriptions for MULTISIG_CONSTRAINTS constants as well?

Reply:
@diego-G Please evaluate if the descriptions are ok and reaffirm @yatki so that he can use the following description.

MULTISIG_CONSTRAINTS.LIFETIME.MINIMUM - Minimum time frame in which the Multisignature transaction will exist in the memory before the transaction is confirmed

MULTISIG_CONSTRAINTS.LIFETIME.MAXIMUM - Maximum time frame in which the Multisignature transaction will exist in the memory before the transaction is confirmed

MULTISIG_CONSTRAINTS.KEYSGROUP.MIN_ITEMS - Minimum number of keys required to sign the Multisignature transaction in the keys group

MULTISIG_CONSTRAINTS.KEYSGROUP.MAX_ITEMS - Maximum number of keys required to sign the Multisignature transaction in the keys group